File: mvpLibrary/src/main/java/cn/cbsd/mvplibrary/net/SoapNetConfig.java
package cn.cbsd.mvplibrary.net;

/**
 * 当前类注释:
 * Author: zhenyanjun
 * Date  : 2018/1/19 11:16
 */

public class SoapNetConfig {

    private static SoapProvider sProvider = null;

    public static void registerProvider(SoapProvider provider) {
        sProvider = provider;
    }

    public static SoapProvider getProvider() {
        return sProvider;
    }
}

File: mvpLibrary/src/main/java/cn/cbsd/mvplibrary/net/SoapNetConfig.kt
package cn.cbsd.mvplibrary.net

/**
 * 当前类注释:
 * Author: zhenyanjun
 * Date  : 2018/1/19 11:16
 */

object SoapNetConfig {

    var provider: SoapProvider? = null

    fun registerProvider(provider: SoapProvider) {
        this.provider = provider
    }
}
